tasks/update.yml

```diff
@@ -1,13 +1,22 @@
 ---
-- name: Run database updates.
-  command: "{{ drush_path }} updatedb -y"
+
+- name: Detect Drupal version.
+  command: drush status drupal-version --format=list
```

Comment:
This should use `{{ drush_path }}` as well

Reply:
This was resolved in 0185c10